storage/cloud.py

```diff
@@ -1032,6 +1034,15 @@ def get_direct_download_url(
                 path, request_ip, expires_in, requires_cors, head, **kwargs
             )
 
+        if kwargs:
+            namespace = kwargs.get("namespace")
+            if namespace is not None:
+                for key in self.cloudfront_distribution_org_overrides:
```

Comment:
if this is a `dict` then you don't need to loop over the keys, you can directly index the name like `cloudfront_distribution_org_overrides[namespace]`

Reply:
I will rework this.